### PR TITLE
Key binding cleanup

### PR DESCRIPTION
**Pull request by slipher**

### CODE REVIEW

File: src/engine/sys/sdl_input.cpp [hunk 1]
```diff
@@ -1032,9 +1032,8 @@ static void IN_ProcessEvents( bool dropInput )
 					Key kKeycode = IN_TranslateSDLToQ3Key( &e.key.keysym, true );
 					bool consoleFound = false;
 					for (Key k: {kScan, kKeycode} ) {
-						if ( IN_IsConsoleKey( k ) && !keys[ Key(K_ALT) ].down) {
+						if ( IN_IsConsoleKey( k ) ) {
```
Comment (DolceTriade) [inline src/engine/sys/sdl_input.cpp:1035]: This is useful, it allows you to type in backticks and tilde into the console for any reason.

File: src/engine/client/cl_keys.cpp [hunk 2]
```diff
@@ -567,16 +573,15 @@ void CL_KeyEvent( const Keyboard::Key& key, bool down, unsigned time )
 	}
 #endif
 
-	// console key is hardcoded, so the user can never unbind it
-	if ( key == Key(K_CONSOLE) || ( keys[ Key(K_SHIFT) ].down && key == Key(K_ESCAPE) ) )
+	// console key combination is hardcoded, so the user can never unbind it
```
Comment (DolceTriade) [inline src/engine/client/cl_keys.cpp:576]: what is cl_consoleKeys? Is it an additional way to open the console?

**Review by DolceTriade — CHANGES_REQUESTED** [review]

File: src/engine/client/cl_keys.cpp (hunk 2 above)
Comment (DolceTriade) [reply src/engine/client/cl_keys.cpp:576]: Ah nvm, this is actually for the shift-esc console bind.

File: src/engine/sys/sdl_input.cpp (hunk 1 above)
Comment (slipher) [reply src/engine/sys/sdl_input.cpp:1035]: Does it really? I don't know any OS where you can enter text while Alt is held.

Comment (DolceTriade) [reply src/engine/sys/sdl_input.cpp:1035]: I can with Linux

Comment (bmorel) [reply src/engine/sys/sdl_input.cpp:1035]: I confirm I can type `²` in console on linux. As for backticks, french layout requires the use of the `alt gr` key anyway, so I can't really tell.

**Review by DolceTriade — APPROVED** [review]

File: src/engine/sys/sdl_input.cpp (hunk 1 above)
Comment (slipher) [reply src/engine/sys/sdl_input.cpp]: It works for me too. I thought I had tested this on Linux and it didn't work. So I put it back.

**Pull request merged**